mungegithub/features/repo-updates.go

```diff
@@ -352,3 +358,16 @@ func (o *RepoInfo) Reviewers(path string) sets.String {
 	}
 	return peopleForPath(path, o.reviewers, false, o.EnableMdYaml)
 }
+
+// IsApproverlessFile takes a path to a file as input and returns a bool
+// indicating whether the specified file does NOT require approval
+func (o *RepoInfo) IsApproverlessFile(path string) bool {
```

Comment:
this is currently checking at the directory level and not for specific files.  Seems useful for things like vendor and generated dirs but less so for individual files that should be skipped (e.g. the makefile in the root dir)